packages/medusa/src/api/routes/admin/products/index.js

```diff
@@ -8,6 +8,11 @@ export default app => {
 
   route.post("/", middlewares.wrap(require("./create-product").default))
   route.post("/:id", middlewares.wrap(require("./update-product").default))
+  route.get("/types", middlewares.wrap(require("./list-types").default))
+  route.get(
+    "/tag-usage",
+    middlewares.wrap(require("./list-tag-usage-count").default)
+  )
```

Comment:
Think we should be able to do something along these lines instead: 
```
GET /tags?sort=usage_count
```

If this is too complicated to implement (could imagine it requires some subqueries in `tagRepo.find`) let's just remove it until we find a better pattern for querying with subqueries.

Reply:
The usage count query was tricky enough, so I say, let's wait till we need tag listing in more ways than now.